client/pages/homepage/carousel-component.js

```diff
@@ -0,0 +1,50 @@
+const CarouselComponent = {
+  template: `
+  <div class="homepage__carousel">
+      <h2 class="carousel__title">see what we have achieved as HYF community!</h2>
+      <h3 class="carousel__subtittle">and many more projects to build...</h3>
+      <div class="container">
+        <div id="carouselExampleControls" class="carousel slide" data-bs-ride="carousel">
+          <div class="carousel-inner">
+            <div class="carousel-item active">
+              <img class="d-block w-100" src="/images/homepage/projectcard_class13-14_handandpaw.png" alt="First slide">
+              <h2 class="slide__tittle">hand and paw / class 13-14</h2>
+              <p class="slide__description">Hand and paw makes animal adoption easier <br> for people, shelters and animals.</p>
+            </div>
+            <div class="carousel-item">
+              <img class="d-block w-100" src="/images/homepage/projectcard_class13-14_readeer.png" alt="Second slide">
+              <h2 class="slide__tittle">readeer / class 13-14</h2>
+              <p class="slide__description">Readeer promotes reading and sharing <br> books among children and parents.</p>
+            </div>
+            <div class="carousel-item">
+              <img class="d-block w-100" src="/images/homepage/projectcard_class11-12_just4giving.png" alt="Third slide">
+              <h2 class="slide__tittle">just4giving / class 11-12</h2>
+              <p class="slide__description">Just4Giving connect those who wants <br> to share with others in need.</p>
+            </div>
+            <div class="carousel-item">
+              <img class="d-block w-100" src="/images/homepage/projectcard_class11-12_helpy.png" alt="Fourth slide">
+              <h2 class="slide__tittle">helpy / class 11-12</h2>
+              <p class="slide__description">Helpy is a sharing platform, its goal is to link those people who need help with their groceries to volunteers who are willing to help.</p>
+            </div>
+            <div class="carousel-item">
+              <img class="d-block w-100" src="/images/homepage/projectcard_class9-10_hearmystory.png" alt="Fifth slide">
+              <h2 class="slide__tittle">hearmystory / class 9-10</h2>
```

Comment:
I would style this to something slightly smaller (also the P that follows

Reply:
Ok thanks

Reply:
You are right about the size of the h2 and the paragraph, but if you see the design, you will notice that are smaller and the slides too. So I pretend to fix that at the end because I think we have to finish Projects and User Profile Pages for backend team to connect them and nobody is working in those pages.
I know we are supposed to push to development when the page is completely done but, if it's the case, header and footer still have issues that I think I can fix at the end too. 
I want to see at least the homepage 100% responsive, but we need to finish those 2 pages this week. I would say for next Tuesday and remember we will have Monday and the same Tuesday.

Likewise, I know you understand my concerns because we talked about that last Wednesday in the call. So, I think is a good argument to merge it to development like this and move on. I promise I will adjust any margin, padding or font size when we see all the pages finished, and I mean the frontend work.

Reply:
Ok for me

Reply:
Your suggestions were accepted and applied @dirk-jacobs. I hope homepage can be merged now